ui/app/templates/components/secret-list-header.hbs

```diff
@@ -13,8 +13,8 @@
           </a>
         </li>
       {{/key-value-header}}
-    {{/p.top}}
-    {{#p.levelLeft}}
+    </p.top>
+    <p.levelLeft>
```

Comment:
Idle thought: Should these yielded components now be yielded in CapitalCase? 🤔

Reply:
Hmm - I'm pretty ¯\_(ツ)_/¯ regarding this - could go either way. If it's just a single character anyway, I'm not sure it adds much because really you're accessing state on the contextual component, not a new component.